src/components/bannerMessages/bannerMessages.js

```diff
@@ -69,51 +50,21 @@ const BannerMessages = ({ messages, useGetAppMessages: useAliasGetAppMessages })
 /**
  * Prop types.
  *
- * @type {{useGetAppMessages: Function, messages: Array}}
+ * @type {{useBannerMessages: Function, useRemoveBannerMessages: Function}}
  */
 BannerMessages.propTypes = {
-  messages: PropTypes.arrayOf(
-    PropTypes.shape({
-      id: PropTypes.string.isRequired,
-      title: PropTypes.node.isRequired,
-      message: PropTypes.node.isRequired,
-      variant: PropTypes.oneOf([...Object.values(AlertVariant)])
-    })
-  ),
-  useGetAppMessages: PropTypes.func
+  useBannerMessages: PropTypes.func,
+  useRemoveBannerMessages: PropTypes.func
 };
 
 /**
  * Default props.
  *
- * @type {{useGetAppMessages: Function, messages: Array}}
+ * @type {{useBannerMessages: Function, useRemoveBannerMessages: Function}}
  */
 BannerMessages.defaultProps = {
-  messages: [
-    {
-      id: 'cloudigradeMismatch',
-      title: translate('curiosity-banner.dataMismatchTitle'),
-      message: translate(
```

Comment:
required:
we no longer need the locale strings associated with this messaging, remove/clean up related strings